block/manager.go

```diff
@@ -226,6 +234,12 @@ func (m *Manager) Start(ctx context.Context) error {
 	/* ----------------------------- full node mode ----------------------------- */
 	if !isProposer {
 
+		// update latest finalized height
+		err = m.updateLastFinalizedHeightFromSettlement()
```

Comment:
why do we need to update it explicitly if it's gonna be updated on the next event anyway?

Reply:
because otherwise on starting a new node (no previous finalized events seen), or when nodes been down for a while, it may start validating already finalized heights when not necssary